ci/docker_ci.sh

```diff
@@ -136,7 +154,9 @@ if [[ "${AZP_BRANCH}" != "${MAIN_BRANCH}" ]] &&
   exit 0
 fi
 
-docker login -u "$DOCKERHUB_USERNAME" -p "$DOCKERHUB_PASSWORD"
+if [[ -z "$DOCKER_CI_DEBUG" || -n "$DOCKER_CI_DEBUG_LOGIN" ]]; then
```

Comment:
```suggestion
if [[ -z "$DOCKER_CI_DEBUG" ]]; then
```

this is not necessary if you want to debug with a real build to a fake repo with fake context - just dont set the debug var